jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -139,6 +139,12 @@ public WebServer updateWebServer(final UpdateWebServerRequest anUpdateWebServerC
                                      final User anUpdatingUser) {
         anUpdateWebServerCommand.validate();
 
+        WebServer originalWebServer=getWebServer(anUpdateWebServerCommand.getId());
```

Comment:
Please use spaces in between operators e.g. '=', '&&' etc for consistency, it has been that way for most projects. Take the case of spring boot https://github.com/spring-projects/spring-boot/blob/master/spring-boot/src/main/java/org/springframework/boot/diagnostics/FailureAnalyzers.java